sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -407,6 +407,7 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
 
   test("create a managed table with the existing non-empty directory") {
     withTable("tab1") {
+      Utils.createDirectory(spark.sessionState.conf.warehousePath)
```

Comment:
Why do we need to create the directory first?

Reply:
As per line 414 it is trying to create empty directory under warehouse path,this statement will fail as parent folder(ware house) does not exists

Reply:
If we do not delete the directory for each test case, this line is not needed.